webapp/crashstats/crashstats/views.py

```diff
@@ -332,8 +333,10 @@ def buginfo(request, signatures=None):
     bug_ids = form.cleaned_data["bug_ids"]
 
     bzapi = models.BugzillaBugInfo()
-    result = bzapi.get(bug_ids)
-    return result
+    try:
+        return bzapi.get(bug_ids)
+    except RetryError:
+        return {"error": "Max retries exceeded with Bugzilla."}, 500
```

Comment:
This return is a structure that `utils.json_view` recognizes and will split into "response, status".